src/developer/web-api/tracker.md

```diff
@@ -2227,6 +2227,8 @@ An example of a json response:
 The change log type can be `CREATE`, `UPDATE`, or `DELETE`.
 `CREATE` and `DELETE` will always hold a single value: the former shows the current value, and the latter shows the value that was deleted. UPDATE will hold two values: the previous and the current.
 
+Change logs are enabled by default and can be configured in `dhis.conf`, as explained [here](../../sysadmin/reference/logging.md#install_changelog)
```

Comment:
It's a bit odd to read this at the end but maybe that's because there is no "changelog" section 🤷🏻 This sounds more like something I would expect at the start of a feature.